packages/near-swap-provider/lib/NearSwapProvider.ts

```diff
@@ -0,0 +1,157 @@
+import { SwapProvider, SwapParams, near } from "@liquality/types";
+
+import Provider from "@liquality/provider";
+import { PendingTxError, TxNotFoundError } from "@liquality/errors";
+import { toNearTimestampFormat, parseReceipt } from "@liquality/near-utils";
+import BigNumber from "bignumber.js";
+import { transactions } from "near-api-js";
+
+import Bytecode from "./bytecode";
+
+const CONTRACT_CODE = "wnGY4a+YYkfbkBqYqyhdQjjtHRl9Auyh7yxCGtXL8JI=";
+
+const ABI = {
+  init: { method: "init", gas: "10000000000000" },
+  claim: { method: "claim", gas: "10000000000000" },
+  refund: { method: "refund", gas: "8000000000000" },
+};
+
+export default class NearSwapProvider
+  extends Provider
+  implements Partial<SwapProvider> {
+  createSwapScript() {
+    return Bytecode;
+  }
+
+  async initiateSwap(
+    swapParams: SwapParams
+  ): Promise<near.NormalizedTransaction> {
```

Comment:
This doesn't seem right. The normalized transaction should be the standard used everywhere `types/transaction`
to specify the underlying object that goes into _raw you can you `Transaction<near.InputTransaction>`